### DISCUSSION TITLE
Not Receiving My Education Benefits

### DISCUSSION THREAD

**Discussion by sylvanding** [community/community#132024] [GitHub Education]
### Hi everyone,

Hi~ My student application has been approved which was submitted on Jul 8th. However, I have not received my education benefits. Could you please assist me in resolving this matter? Thank you very much for your help.

**Comment by naveenalla3000** [comment]
if it is approved correctly, within 2-4 days you will receive. If any incorrect submission make sure that you are submitting correct information(name of collage, link college mail with GitHub, proper Id card photo).
mark it as answer if it helpful.

**Comment by Ali01129** [comment]
if it is approved just go to copilot and select a plan and then they will tell you that you have student account and then it will be free to use.
